autoload/neomake/compat.vim

```diff
@@ -174,7 +174,7 @@ elseif neomake#has_async_support()  " Vim-async.
             if argv[0:len(prefix)-1] == prefix
                 return argv
             endif
-            return prefix.neomake#utils#shellescape(argv)
+            return argv
```

Comment:
`prefix` should be kept here, no?
It also break `tests/compat.vader` (you can try running it with `make tests/compat.vader`).

Everything after e.g. `sh -c` (or `cmd.exe /c`) should be shellescaped I think.